src/main/java/com/ap3xtbh/raidbot/util/Config.java

```diff
@@ -0,0 +1,99 @@
+package com.ap3xtbh.raidbot.util;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Arrays;
+
+
+/**
+ * Pull data from bot config json
+ */
+public class Config {
+    private final Logger log = LoggerFactory.getLogger(Config.class);
+
+    private String token = null;
+    private String ownerID = null;
+    private final String[] coOwnerIDs = new String[2];
+    private String guildID = null;
+    private String prefix = null;
+
+    public String getToken() {
+        return token;
+    }
+
+    public String getOwnerID() {
+        return ownerID;
+    }
+
+    public String[] getCoOwnerIDs() {
+        return coOwnerIDs;
+    }
+
+    public String getGuildID() {
+        return guildID;
+    }
+
+    public String getPrefix() {
+        return prefix;
+    }
+
+    public void load() {
```

Comment:
tbh if any of this data fails to load, we should probably just throw an exception and/or abort the bot initialization, since most of this is necessary for it to work properly.